CONTRIBUTING.md

```diff
@@ -78,17 +78,17 @@ Check out our dedicated [Bevy Organization document](/docs/the_bevy_organization
 
 Our merge strategy relies on the classification of PRs on two axes:
 
-* How controversial are the design decisions
-* How complex is the implementation
+* How controversial are the design decisions.
+* How complex is the implementation.
```

Comment:
These are questions, so if we want punctuation here, the sentences should end with a question mark.

Reply:
Or rephrase as e.g. `How complex the implementation is.`